prescient.el

```diff
@@ -79,15 +79,14 @@ This only has an effect if `prescient-persist-mode' is enabled."
   :type 'file)
 
 (defvar prescient--filter-method-custom-type
-  '(choice
+  '(set
     (const :tag "Literal" literal)
-    (const :tag "Initialism" initialism)
-    (const :tag "Literal and initialism" literal+initialism)
     (const :tag "Regexp" regexp)
+    (const :tag "Initialism" initialism)
     (const :tag "Fuzzy" fuzzy))
```

Comment:
This Custom type should also handle the case where only a single symbol is provided.

Reply:
I don't understand. This custom type will let you set a single symbol. It will make it a list containing just that symbol, and in our case a list of a single symbol is the handled the same as the symbol itself.

Reply:
Huh, okay. I guess you're right, it's not important to allow the user to set the value to a single symbol—and allowing that would probably just make the user interface more complicated.